### ISSUE TITLE
timex requires specific version of tzdata

### ISSUE THREAD

**Issue by brianmay** [bitwalker/timex#634]
https://github.com/bitwalker/timex/blob/master/mix.exs#L59

This means if I try to update tzdata to 1.1.0, I get errors:

```
Failed to use "tzdata" (version 1.1.0) because
  /home/circleci/project/robotica_plugins/mix.exs requires ~> 1.1.0
  calendar (version 1.0.0) requires ~> 0.5.20 or ~> 0.1.201603 or ~> 1.0
  timex (version 3.6.3) requires ~> 0.1.8 or ~> 0.5 or ~> 1.0.0
```

It looks like calendar has a similar problem. Although I might try to remove that dependency.

**Comment by Hermanverschooten** [comment]
I am currently using `override: true` on `tzdata`, without any issues.

**Comment by arjan** [comment]
Will be fixed in #632
